From 1ee0cffab20f91ff556827382c0f2e8c1a579372 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Thu, 5 May 2022 09:22:21 +0200
Subject: [PATCH] Replace tabs with spaces in domain_all_settings.xml

The xml renderer stopped rendering tabs and now uses spaces instead, which made
the test fail.
---
 spec/unit/templates/domain_all_settings.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/spec/unit/templates/domain_all_settings.xml b/spec/unit/templates/domain_all_settings.xml
index eaa033a..cac4b50 100644
--- a/spec/unit/templates/domain_all_settings.xml
+++ b/spec/unit/templates/domain_all_settings.xml
@@ -112,12 +112,12 @@
     </hostdev>
     <hostdev mode='subsystem' type='pci' managed='yes'>
       <source>
-	<address domain='0x0001' bus='0x03' slot='0x00' function='0x0'/>
+        <address domain='0x0001' bus='0x03' slot='0x00' function='0x0'/>
       </source>
     </hostdev>
     <hostdev mode='subsystem' type='pci' managed='yes'>
       <source>
-	<address domain='0x0002' bus='0x04' slot='0x00' function='0x0'/>
+        <address domain='0x0002' bus='0x04' slot='0x00' function='0x0'/>
       </source>
       <address type='pci' domain='0x0000' bus='0x01' slot='0x01' function='0x0'/>
     </hostdev>
-- 
2.35.1

